tools/build/libwebrtc/mrwebrtc.ps1

```diff
@@ -95,6 +119,13 @@ function Install-GoogleRepository {
         throw
     }
 
+    # Delete sources not needed; this prevent security alerts on unused components,
+    # and makes the overall checkout size smaller.
+    Clear-PostCheckout -SourceFolder $libwebrtcFolder
+
+    # Remove depot_tools/external_bin/gsutil
+    Remove-Item -Path $(Join-Path $externalFolder "depot_tools/external_bin/gsutil" -Resolve) -Force -Recurse | Out-Null
```

Comment:
nit: specify that this is done because of security alerts too (or simply group with line above).

Reply:
Grouped with above.